lib/search_form.ex

```diff
@@ -96,6 +97,16 @@ defmodule Prismic.SearchForm do
     end
   end
 
+  def set_orderings(%SearchForm{} = search_form, nil) do
```

Comment:
I think it would be better code organization to make this function private and test it through `everything_search_form`. Not a big deal though.

Reply:
I thought about that too @newtrat . 

To make this function private and test it through `everything_search_form` I think would not be possible 🤔 b/c we added the ability to pass a custom order without always being the default one. So, somehow we must pass this order to the submit form.

Being private, the call of this function would be inside `SearchForm.submit/1` and we would pass the order as a parameter too this function b/c I added the possibility of the user passes the order he prefers. 

It's an easy change but I decided to avoid changing the signature of submitting function to not depend on receiving another param and let this be a responsibility of another function. 

Both ways seem to be simple for me, But I think I'll keep the way it is. But that was a good observation. Thanks ✨ . 

Sorry for the time a took to answer this simple comment.